BrowserKit/Sources/WebEngine/WKWebview/WKEngineWebView.swift

```diff
@@ -28,6 +28,7 @@ protocol WKEngineWebView: UIView {
     var estimatedProgress: Double { get }
     var canGoBack: Bool { get }
     var canGoForward: Bool { get }
+    var hasOnlySecureContent: Bool { get }
```

Comment:
I wouldn't add this now since it's not used + this will break the mock of web view in the WebEngineTests. We can integrate changes in the WebEngine package with its own task, namely [FXIOS-8088](https://mozilla-hub.atlassian.net/browse/FXIOS-8088)